src/ViewRendererBundle/Bridge/Chameleon/Migration/Script/update-1539324183.inc.php

```diff
@@ -0,0 +1,38 @@
+<h1>update - Build #1539324183</h1>
+<h2>Date: 2018-10-12</h2>
+<div class="changelog">
+    - #117: Only use necessary additional cache settings
+</div>
+<?php
+
```

Comment:
I think we should add a short notice that informs the user of this change, so that is doesn't come by surprise.

Reply:
ok